Browser_IDE/stylesheet.css

```diff
@@ -57,9 +57,7 @@ button:disabled {
     cursor: wait;
 }
 
-button:hover:enabled {
-    background-color: #404040;
-}
```

Comment:
This causes none of the other buttons in the IDE to highlight when hovered, would be better to leave this as-was.